### PR TITLE
Closes #335

### LINKED ISSUE #335: 🐛 [Bug report]: Expanding-card play's active card is too wide on small screen.
**Describe the bug**
On small screen, the active card of Expanding-card play is too wide. it's a small tweak if someone looking to get started can fix it. 

Solution:
the width of .active class needs to be fixed on small screen

**To Reproduce**
1. Go to https://reactplay.io/plays/expanding-cards/
2. resize the screen to smaller screen
3. look at the active card
4. active card looks too wide

**Expected behavior**
active card should be well aligned with other cards. 

**Screenshots**

expanding-card

**Desktop (please complete the following information):**
 - OS: ubuntu
 - Browser brave

### PR DESCRIPTION
**Pull request by lpizzinidev**
Fixes #335 

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

# How Has This Been Tested?

- [x] I have performed a self-review of my own code
- [x] My changes generate no new warnings

### CODE REVIEW

**Comment by atapas** [comment]
@Deepak8717 Please review the changs.. You can also check the preview URL

File: src/plays/expanding-cards/ExpandingCards.css [hunk 1]
```diff
@@ -26,8 +26,10 @@
   -webkit-backface-visibility: hidden;
 }
 
-.card-container .active {
-  width: 600px;
+@media only screen and (min-width: 600px) {
```
Comment (Deepak8717) [inline src/plays/expanding-cards/ExpandingCards.css:29]: unnecessary media query. There is already a section where all media queries lie at the bottom. you either need to 
```
.card-container .active {
    width:100%;
    max-width:600px
  }
or just add the width:100% in media query of same class that has been declared at the bottom.

**Review by Deepak8717 — COMMENTED** [review]
Please check the comments. We don't need additional media queries. Just change 
```width:600px``` to 

```
 width:100%;
 max-width:600px;
   ```

File: src/plays/expanding-cards/ExpandingCards.css (hunk 1 above)
Comment (lpizzinidev) [reply src/plays/expanding-cards/ExpandingCards.css:29]: 👍 I will make the adjustment. Thanks for the feedback.

**Comment by Deepak8717** [comment]
@atapas looks good.

**Review by Deepak8717 — APPROVED** [review]

**Review by atapas — APPROVED** [review]

**Comment by atapas** [comment]
@all-contributors please add @lpizzinidev for Code

**Pull request merged**